servlet.py

```diff
@@ -1034,6 +1037,7 @@ def sanity_check():
     parser.add_argument('-wu', '--wiki-username', help="Apertium Wiki account username for SuggestionHandler", default=None)
     parser.add_argument('-b', '--bypass-token', help="ReCAPTCHA bypass token", action='store_true')
     parser.add_argument('-rs', '--recaptcha-secret', help="ReCAPTCHA secret for suggestion validation", default=None)
+    parser.add_argument('-mp', '--max-processes', help="Max no. of processes to be yielded", default=None)
```

Comment:
"max processes" is too general (does it refer to tornado processes? total spawned children? etc.). This is not even max total child processes, or even max document translation processes; it's max document translation pipelines (each pipeline having many child processes). A better name might be `'--md', '--max-doc-pipes'`